src/components/Transfer/Submit.js

```diff
@@ -66,45 +119,19 @@ class Submit extends React.Component {
 			const { amount, recipient, message } = this.props;
 			const amountAsFloat = parseFloat(amount.replace(',', '.'));
 
-			dispatch(
-				Config.spinner({
-					visible: true,
-					textContent: t('transfering'),
-				})
+			Alert.alert(
+				t('confirm_title'),
+				t('confirm_message', { amount: floatToEuro(amountAsFloat), name: recipient.name }),
+				[
+					{ text: _('cancel'), onPress: () => this.cancel(dispatch) },
+					{
+						text: t('confirm'),
+						onPress: () => this.accept(dispatch, amountAsFloat, recipient, message, navigation),
```

Comment:
No no no no no never !
You are passing `dispatch` which is an injected prop. Fetch it in each method: accept and cancel
Moreover, `recipient` and `message` are also props but here as they are part of the accepting point, we can pass them.